nengo_gui/static/nengo.js

```diff
@@ -39,7 +39,9 @@ Nengo.create_websocket = function(uid) {
     } else{
         var ws_proto = 'ws:';
     }
-    var ws_url = ws_proto + '//' + parser.host + '/viz_component?uid=' + uid;
+    var ws_url = ws_proto + '//' + parser.host + window.location.pathname +
+        'viz_component?uid=' + uid;
+    console.log(ws_url)
```

Comment:
I assume this console.log should be removed?  :)

Reply:
yeah